keps/sig-storage/1472-storage-capacity-tracking/README.md

```diff
@@ -287,6 +273,11 @@ so nothing changes for existing CSI driver deployments.
 
 ### API
 
+The API is a new builtin type. This approach was chosen instead of a
+CRD because kube-scheduler will depend on this API and ensuring that a
+CRD gets installed and updated together with the core Kubernetes is
+still an unsolved problem.
+
```

Comment:
nit: fix indentation for types defined below

Reply:
Fixed.